.github/workflows/node-build.yml

```diff
@@ -0,0 +1,309 @@
+        with:
+          files: |
+            **/generated/graphql.*
+
+      - name: fail if GraphQL was generated
+        if: steps.verify-changed-files.outputs.files_changed == 'true'
+        run: exit 1
+  
+  codeql-analyze:
+    runs-on: ubuntu-latest
+    needs: prerequisite
+    timeout-minutes: 5
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
+        config:
+          - './.github/codeql/source.yml'
+          - './.github/codeql/tests.yml'
+    steps:
+     - uses: actions/checkout@v4
+     - uses: ./.github/workflows/rafiki/env-setup
+     - name: Initialize CodeQL
+       uses: github/codeql-action/init@v3
+       with:
+         languages: ${{ matrix.language }}
+         config-file: ${{ matrix.config }}
+     - name: Perform CodeQL Analysis
+       uses: github/codeql-action/analyze@v3
+
+  node-build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    needs: [auth, backend, frontend, token-introspection, mock-account-servicing-entity, graphql, codeql-analyze]
+    steps: 
+      - uses: actions/checkout@v4
+      - uses: ./.github/workflows/rafiki/env-setup
+      - run: pnpm build
+
+  version-generator:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref_name, 'release/v') 
+    outputs:
+      version: ${{ steps.version-generator.outputs.NEW_VERSION }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: true
+      - name: Configure git
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+      - id: version-generator
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          VERSION_PREFIX=$(echo "${{ github.ref_name }}" | sed 's|release/||')
+          read major minor patch pre_release <<< $(echo "$VERSION_PREFIX" | awk -F'[.v-]' '{print $2, $3, $4, $5}')
+
+          version_search="v$major.$minor.*"
+
+          if [ -n "$pre_release" ]; then
+              version_search="$version_search-$pre_release"
+          fi
+          echo "VERSION_SEARCH: $version_search"
+          VERSION_PREFIX=$(git tag -l $version_search --sort=-taggerdate | head -n 1)
+
+          if [ -n "$VERSION_PREFIX" ]; then
+            read major minor patch pre_release <<< $(echo "$VERSION_PREFIX" | awk -F'[.v-]' '{print $2, $3, $4, $5}')
+            patch=$((patch + 1))
+          fi
+
+          NEW_VERSION="v${major}.${minor}.${patch}"
+
+          if [ -n "$pre_release" ]; then
+            NEW_VERSION="$NEW_VERSION-${pre_release}"
+          fi
+
+          echo "NEW_VERSION=$NEW_VERSION" >> $GITHUB_OUTPUT
+          echo "New version will be: $NEW_VERSION"
+
+          git tag -a $NEW_VERSION -m "$NEW_VERSION"
+          git push origin $NEW_VERSION
+  
+  docker-build:
```

Comment:
Do I just not see it or are we not doing the multi-platform builds anymore?

Reply:
Nice catch. I indeed forget to add also platforms in docker build step.